mathutil.go

```diff
@@ -631,6 +631,28 @@ func Min(a, b int) int {
 	return b
 }
 
+// MaxVal returns the largest argument passed.
+func MaxVal(val int, vals ...int) int {
```

Comment:
I intentionally did not replace the existing functions, in case they were being used in performance-sensitive applications.

Reply:
Good idea, thanks.